### PR TITLE
Starting new rounds

### PR DESCRIPTION
**Pull request by bumblebot1**

### CODE REVIEW

File: src/server/services/room-data-service.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	    },
+	    possibleAnswers: {},
+	    finalAnswerList: {}
+	}
+}
+
+function newRoundData(oldData) {
+	Object.assign(oldData,{
+		answering: false,
+    	submitting: false,
```
Comment (kenneth-gray) [inline src/server/services/room-data-service.js]: Strange indentation around here

File: src/server/services/room-data-service.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        	letters: [],
+        	consonantNum: 0,
+        	vowelNum: 0,
+        	disableConsonant: false,
+        	disableVowel: false
+    	},
+    	possibleAnswers: {},
+    	finalAnswerList: {}
+	});
+	return oldData;
```
Comment (kenneth-gray) [inline src/server/services/room-data-service.js]: Doesn't the above Object.assign need to be assigned to newData and then newData returned?

File: src/server/services/room-data-service.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+module.exports = {
+	initialRoomData,
+	newRoundData
+}
+
+function initialRoomData(user) {
+	return {
+		creator: {
```
Comment (kenneth-gray) [inline src/server/services/room-data-service.js]: Indentation?

File: src/server/cloak-server.js [hunk 4]
```diff
@@ -473,10 +473,24 @@ function sendChosenWordList(room, answers){
             }
         }
     }
-
+    room.data.roundResults = toSend;
     if(roomMembers.length === Object.keys(toSend).length){
```
Comment (kenneth-gray) [inline src/server/cloak-server.js]: I have no idea what this means? Could it be a function maybe to explain?

File: src/server/services/room-data-service.js (hunk 1 above)
Comment (bumblebot1) [inline src/server/services/room-data-service.js]: github seems to mess it up; on both our machines it looks fine :confused:

File: src/server/cloak-server.js (hunk 4 above)
Comment (JamieZafar) [inline src/server/cloak-server.js]: would assigning object.keys(toSend).length to an appropriately named variable suffice?

**Comment by thullSL** [comment]
:shipit:

**Pull request merged**